### PR TITLE
Set status merge

### PR DESCRIPTION
**Pull request by wreiske**

### CODE REVIEW

**Comment by wreiske** [comment]
@Prateek93a do you think you could resolve the merge conflicts? Thanks.

**Comment by Prateek93a** [comment]
@wreiske  Done.

File: app/views/SidebarView/index.js [hunk 1]
```diff
@@ -142,6 +154,17 @@ class Sidebar extends Component {
 		});
 	}
 
+	setStatusText = async(statusText) => {
+		const result = await RocketChat.saveUserProfile({ statusText }, {});
+		if (result.success) {
+			EventEmitter.emit(LISTENER, { message: I18n.t('Status_saved_successfully') });
```
Comment (wreiske) [inline app/views/SidebarView/index.js:160]: Is there a reason why these are not consistent? "Status_save..." vs "error-could".

File: app/views/SidebarView/index.js [hunk 2]
```diff
@@ -142,6 +154,17 @@ class Sidebar extends Component {
...
+		} else {
+			EventEmitter.emit(LISTENER, { message: I18n.t('error-could-not-change-status') });
```
Comment (wreiske) [inline app/views/SidebarView/index.js:162]: See the above comment about consistency in the i18n string. We should make these consistent.

Comment (wreiske) [reply app/views/SidebarView/index.js:162]: Actually, scrap that. It looks like it is consistent with what is in the i18n file. Nevermind.

File: app/views/SidebarView/index.js (hunk 1 above)
Comment (wreiske) [reply app/views/SidebarView/index.js:160]: Actually, scrap that. It looks like it is consistent with what is in the i18n file. Nevermind.

**Review by wreiske — COMMENTED** [review]
I still need to test these changes. I will get it tested in the next few days. Thanks.

**Comment by Prateek93a** [comment]
Thank you. Hope you liked my work.

**Comment by wreiske** [comment]
@Prateek93a There is a bug when you update your status on desktop, the modal window text doesn't change.

1. Change status on mobile app.
2. Change status on desktop
3. Tap to change status on mobile app to see the original status message in the modal.

**Pull request merged**